components/kyma-environment-broker/internal/orchestration/dto.go

```diff
@@ -0,0 +1,12 @@
+package orchestration
+
+import "github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+
+type UpgradeOrchestrationDTO struct {
+	Targets  internal.TargetSpec   `json:"targets"`
+	Strategy internal.StrategySpec `json:"strategy,omitempty"`
+}
+
+type UpgradeOrchestrationResponseDTO struct {
+	OrchestrationID string `json:"operation_id"`
```

Comment:
As this would be returned for each newly created `Orchestration`, consider renaming it to something like:

```suggestion
type OrchestrationSpec struct {
	OrchestrationID string `json:"orchestration_id"`
```

similar to naming convention in `domain` package in KEB

Reply:
sure but OrchestrationSpec conflicts with the package name so I will go with `UpgradeResponse` due to its versatility